### PR TITLE
Delete tab_lang when delete a tab

### PR DESCRIPTION
**Pull request by rubensmussicury**
I see no reasons to leave the tab_lang table records for some tab that does not exist anymore.

### CODE REVIEW

File: classes/Tab.php [hunk 1]
```diff
@@ -150,7 +150,10 @@ public static function initAccess($id_tab, Context $context = null)
 
 	public function delete()
 	{
-		if (Db::getInstance()->execute('DELETE FROM '._DB_PREFIX_.'access WHERE `id_tab` = '.(int)$this->id) && parent::delete())
+	 	if (Db::getInstance()->execute('DELETE FROM '._DB_PREFIX_.'access WHERE `id_tab` = '.(int)$this->id) &&
+			Db::getInstance()->execute('DELETE FROM '._DB_PREFIX_.'tab_lang WHERE `id_tab` = '.(int)$this->id) &&
+			parent::delete())
+		{
```
Comment (gRoussac) [inline classes/Tab.php]: Hi,

Thank you for your PR. I believe your are adding an additional bracket here.

Regards

Comment (rubensmussicury) [inline classes/Tab.php]: In fact. I'm sorry about that. I copied and pasted form my IDE :/

**Comment by guestisp** [comment]
+1

**Comment by vTerenti** [comment]
(note: this is an automated message, but answering it will reach a real human )

Hi! Thank you for your pull request: your contribution to PrestaShop means a lot to us!

We are aware that this PR is still open after all this time, and we want to make sure that your hard work is not forgotten.

We are diving back into pull requests, and we want to work with you in order to get your PR to completion: we need your feedback!<br/>
Could you read [this article](http://build.prestashop.com/news/review-your-pull-requests/) from [the Build devblog](http://build.prestashop.com/), and take the time to review your pull requests, so that we can work with up-to-date information? You don't have to do it all right now, and every little step helps.

Again, thank you very much for this contribution, and for your renewed feedback on it!

**Comment by maximebiloe** [comment]
Hello,

The ObjectModel already takes care of this [here](https://github.com/PrestaShop/PrestaShop/blob/develop/classes/ObjectModel.php#L773)

Regards

**Pull request closed without merging**